app/components/CreateWalletForm/ConfirmSeed/Form.js

```diff
@@ -2,33 +2,50 @@ import React from "react";
 import SeedEntry from "./SeedEntry";
 import "../../../style/CreateWalletForm.less";
 
-const ConfirmSeedForm = ({
-  remainingSeedWords,
-  seedError,
-  setSeedWords,
-  isMatch,
-  isEmpty
-}) => (
-  <div className="confirm-seed">
-    <div className="create-wallet-label">
-      <div className="confirm-seed-label-text">Confirm Seed:</div>
-      <div className="confirm-seed-label-remaining-words">{remainingSeedWords} words remaining</div>
-    </div>
-    <div className="create-wallet-field">
-      <div className="input-form">
-        <form className="input-form-confirm-seed">
-          <SeedEntry label="Seed Entry" onChange={setSeedWords}/>
-        </form>
-      </div>
-      <div className="input-form-error">
-        {seedError
-          ? seedError
-          : isMatch || isEmpty
-            ? null
-            : "*Seeds do not match"}
-      </div>
-    </div>
-  </div>
-);
+class ConfirmSeedForm extends React.Component{
+
+  constructor(props){
+    super(props);
+    this.state = {
+      showPasteWarning : false
+    };
+  }
+
+  handleOnPaste = (e) => {
+    e.preventDefault();
+    this.setState({
+      showPasteWarning : true
+    });
+  }
+
+  render(){
+    const { remainingSeedWords, setSeedWords, isMatch, seedError, isEmpty } = this.props;
+    return (
+      <div className="confirm-seed">
+          <div className="create-wallet-label">
+            <div className="confirm-seed-label-text">Confirm Seed:</div>
+            <div className="confirm-seed-label-remaining-words">{remainingSeedWords} words remaining</div>
+          </div>
+          <div className="create-wallet-field">
+            <div className="input-form">
+              <div className="orange-warning">{!this.state.showPasteWarning
```

Comment:
You should also add the div `orange-warning` inside the statement. If not, the div `orange-warning` will be rendered empty and that's not needed :)

```
{!this.state.showPasteWarning ? null : <div className="orange-warning">*You should not paste your Seeds. Please type it</div>}
```

will be perfect!

Reply:
Agreed, this anti-pattern of empty elements popped up a lot in the older component code and may still lurk in some places.

But react deals with nulls just fine.